src/test/java/org/fundacionjala/sergio/CompleteNumberTest.java

```diff
@@ -0,0 +1,31 @@
+package org.fundacionjala.sergio;
+
+import org.fundacionjala.at15.katas.bankocr.sergio.Splitter;
+import org.fundacionjala.at15.katas.bankocr.sergio.MatrixArch;
+import org.fundacionjala.at15.katas.bankocr.sergio.NumberDef;
+import org.fundacionjala.at15.katas.bankocr.sergio.CompleteNumber;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class CompleteNumberTest {
+
+    String line1 = "    _  _     _  _  _  _  _ ";
+    String line2 = "  | _| _||_||_ |_   ||_||_|";
+    String line3 = "  ||_  _|  | _||_|  ||_| _|";
+
+    String completNumberToCompare = "123456789";
+
+
+    @Test
+    public void itShouldReturnTheCompleteNumber(){
+        Splitter theSpliter = new Splitter(line1,line2,line3);
+        MatrixArch matrix = new MatrixArch(theSpliter);
+        NumberDef number = new NumberDef(matrix);
```

Comment:
The block of code
```
Splitter theSpliter = new Splitter(line1,line2,line3);
MatrixArch matrix = new MatrixArch(theSpliter);
NumberDef number = new NumberDef(matrix);
```
is used in multiple places in different classes, this is a good candidate to be moved to a specific class/method so the functionality can be re-used across all your classes